components/faq.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Document</title>
+    </head>
+    <body>
+        <main>Need Help?
+            <input type="search" name="searchFaq">
+        </main>
+        <h1>FAQS</h1>
+        <section class="question">
```

Comment:
Also use sections to wrap the title of the section (Question X) and the div inside of it.